### DISCUSSION TITLE
SOUNDCLOUD ACCOUNT DELETED WITH NO FEEDBACK. PLEASE HELP SOMEBODY.

### DISCUSSION THREAD

**Discussion by glitcheetah** [community/community#27705] [Other Feature Feedback, Questions, & Ideas]
A cry for help For soundcloud developers and their technical support...

You deleted my Cybart account last night. Please restore it back. I've been collecting music for it for years. Looking for musicians. I posted tracks, there were 35 of my tracks with auditions. This account means a lot to me, I invested a lot in it. Explain the reason for the removal? Am I opening too many tabs at the same time? Did I like too much? Did I do it too suddenly? So I couldn't use the service for two weeks while I was on vacation, and then the desire to listen to music came over me, and after that you block me? Tell me that this is a stupid bot that misinterpreted my actions, and not that you have such evil employees sitting in the office. 
@Júlio Zynger https://github.com/julioz @Sarah Port  https://github.com/seport?tab=repositories @Marc Tudurí https://github.com/marctc @Bernd Louis https://github.com/parkl @Jan Monschke https://github.com/janmonschke

I can’t contact anyone directly and find out the reason for deleting my account, I have no idea how to restore all the information that was stored there for 6 years of use. You can close access to the account, but at least restore it so that I can see my playlists and list of reposts. I have no idea what to do, so I am writing this post here and mention those developers whom I could find in the incomprehensible soundcloud support system.

**Comment by MrSarno** [comment]
You should be able to contact SoundCloud's support team by filling in a ticket from [here](https://help.soundcloud.com/hc/en-us/requests/new?ticket_form_id=%20581048).

There's absolutely no chance their developers, who are from the wrong department, will discuss your account with you publicly on a third-party platform. Please try to refrain from creating obviously off-topic posts here; it's not fair on other community members.
